lib/hammer_cli_katello/content_export.rb

```diff
@@ -44,6 +44,45 @@ def execute
       end
     end
 
+    class GenerateListingCommand < HammerCLIKatello::Command
+      desc _("Generates listing file on each directory of a syncable export. This command "\
+        + "only needs to be used if the export was performed asynchronously or if the listing files"\
+        + " were lost. "\
```

Comment:
The most common reason the listing files might be missing is that hammer didn't have permissions..
```suggestion
        + "only needs to be used if the export was performed asynchronously or if the listing files"\
        + "were not generated at time of export. "\
```